include/emqx_release.hrl

```diff
@@ -29,7 +29,7 @@
 
 -ifndef(EMQX_ENTERPRISE).
 
--define(EMQX_RELEASE, {opensource, "4.3.2"}).
```

Comment:
Hi, @nicklasfrahm 
Thank you for your contribution, but I think this part of the modification should be revoked, and we will update this flag before releasing the new version